.github/workflows/docs.yml

```diff
@@ -19,6 +19,9 @@ jobs:
       - name: Build protobuf
         run: inv protoc
 
+      - name: Install package + deps
+        run: pip install -e .  # Makes sure doc generation doesn't break on client imports etc.
```

Comment:
This is somewhat unrelated, but the build failed because we don't have explicit requirements on client dependencies here. Presumably succeeded before this since dependencies were cached, but now that we needed some new globals/imports from a new synchronicity version things broke